scripts/bigquery_billing_report/bigquery_billing_report.py

```diff
@@ -147,64 +147,52 @@ def print_submission_pricing(ws_namespace, ws_name, query_sub_id, query_workflow
                 workflow_ids_with_no_cost.add(wf_id)
                 continue
 
+            print "|\t.--- Workflow %d of %d: %s (%s)" % (wf_count, wf_total_count, wf_id, workflow_json["status"])
+            wf_count += 1
+            
             workflow_metadata_json = get_workflow_metadata(ws_namespace, ws_name, submission_id, wf_id)
 
             if "calls" not in workflow_metadata_json:
-                # print ws_namespace, ws_name, submission_id, wf_id, workflow_metadata_json
-                continue
-
-            # workflow_metadata_json = workflow_id_to_metadata_json[wf_id]
-            calls_lower_json = dict(
-                (k.split(".")[-1].lower(), v) for k, v in workflow_metadata_json["calls"].iteritems())
-
-            if len(calls_lower_json) == 0:
                 print "\tNo Calls."
```

Comment:
Can this be renamed: "Workflow x was Free!"

Reply:
i guess free is a positive spin on the fact that it failed to run :)